src/Camera.ios.tsx

```diff
@@ -21,7 +21,9 @@ const Camera = React.forwardRef((props: CameraProps, ref: any) => {
     },
   }));
 
-  return <NativeCamera style={{ minWidth: 100, minHeight: 100 }} ref={nativeRef} {...props} />;
+  return (
+    <NativeCamera maxZoom={props.maxZoom ?? 20} style={{ minWidth: 100, minHeight: 100 }} ref={nativeRef} {...props} />
+  );
 });
 
 Camera.defaultProps = {
```

Comment:
We can use `defaultProps` bellow to set 20 by default

```suggestion
    <NativeCamera style={{ minWidth: 100, minHeight: 100 }} ref={nativeRef} {...props} />
  );
});

Camera.defaultProps = {
  maxZoom: 20,
```

Reply:
Good catch. I actually wanted to remove that. maxZoom shouldn't be required in RN, as that means we can't tell if it's undefined (unrestricted) or 20 (restricted).